app/public/service-worker.js

```diff
@@ -0,0 +1,90 @@
+var reviewRedirect, title, options;
+title = 'We Need Your Help!';
+options = {
+	body: '',
+	icon: '/images/logo-192x192.png',
+	sound: '/sounds/alert.wav',
+	actions: [
+			{
+			action: 'review',
+			title: 'Yes!'
+			}
+		],
+	tag: 'help-notification',
+	vibrate: [500,110,500,110,450]
+	};
+function helpNotification(name,id) {
+	if(name=='404')
+		options.body='';
+	else{
+		reviewRedirect='location/rating/'+id+'/5';
+		options.body='Would you like to add a review to '+name+'?';
+	}
+	//self.registration.showNotification(title, options);
+};
+
+function sendNotification() {
+	//check to send here
+	sendMessage({type: 'refresh'});
+	if(options.body!=='')
+		self.registration.showNotification(title,options);
+}
+
+function sendMessageToClient(client, msg){
+    return new Promise(function(resolve, reject){
+        var msg_chan = new MessageChannel();
+
+        msg_chan.port1.onmessage = function(event){
+            if(event.data.error){
+                reject(event.data.error);
+            }else{
+                resolve(event.data);
+            }
+        };
+
+        client.postMessage(msg, [msg_chan.port2]);
+    });
+}
+
+function sendMessage(content){
+	clients.matchAll().then(clients => {
+		clients.forEach(client => {
+			sendMessageToClient(client, content);
+		})
+	})
+}
+
+self.addEventListener('install', function(e) {   
+	e.waitUntil(self.skipWaiting());
+    console.log('Service Worker Installed')
+})  
+
+self.addEventListener('activate', function(e) {
+	e.waitUntil(self.clients.claim())
+    console.log('Service Worker Activated');
+})
+
+self.addEventListener('fetch',function(e) {
+    //console.log('fetching '+e);
```

Comment:
Remove this code that doesn't do anything.  The fetch here doesn't do anything.  I guess you used it for troubleshooting but it doesn't need to get deployed.